README.md

```diff
@@ -1,170 +1,269 @@
+| Flower        | 5556  | Task monitoring dashboard |
+| Infinite Talk | 8001  | AI conversation service   |
+| Zonos Server  | 8002  | AI processing service     |
+
+### System Flow
+
+```mermaid
+sequenceDiagram
+    participant Client
+    participant FastAPI as FastAPI Container
+    participant Redis as Redis (Remote)<br/>redis:6379
+    participant RouterWorker as Router Worker<br/>(local queue)
+    participant InfiniteTalk as infinite_talk Worker<br/>(infinite_talk queue)
+    participant Zonos as zonos Worker<br/>(zonos queue)
+
+    Client->>FastAPI: HTTP Request with task
+    FastAPI->>Redis: Enqueue send_task_to_server<br/>queue: "local"
+
+    Redis->>RouterWorker: Poll task from "local" queue
+    RouterWorker->>RouterWorker: Execute send_task_to_server()<br/>server_name, task_name, payload
+
+    alt Route to infinite_talk
+        RouterWorker->>Redis: app.send_task()<br/>queue: "infinite_talk"
+        Redis->>InfiniteTalk: Poll from "infinite_talk" queue
+        InfiniteTalk->>InfiniteTalk: Process task_name<br/>with payload
+        InfiniteTalk->>Redis: Store result
+        Redis->>RouterWorker: Return task.id
+    else Route to zonos
+        RouterWorker->>Redis: app.send_task()<br/>queue: "zonos"
+        Redis->>Zonos: Poll from "zonos" queue
+        Zonos->>Zonos: Process task_name<br/>with payload
+        Zonos->>Redis: Store result
+        Redis->>RouterWorker: Return task.id
+    end
+
+    RouterWorker->>Redis: Return task.id
+    Redis->>FastAPI: Task result (task.id)
+    FastAPI->>Client: HTTP Response with task.id
+
+    Note over Client,Redis: Polling Phase
+
+    loop Poll for result
+        Client->>FastAPI: GET /task_status/{task_id}
+        FastAPI->>Redis: Check task status/result
+        Redis->>FastAPI: Status (PENDING/SUCCESS/result)
+        FastAPI->>Client: Response (status or result)
+    end
+```
+
+---
 
-### Development Tools
+## Advanced Usage
 
-- **Celery Worker**: Background task processing for heavy operations
-- **Flower** (Port 5556): Real-time monitoring of Celery tasks
+### Docker Compose Profiles
 
-## Service Profiles
+Run different combinations of services based on your needs:
 
-The application uses Docker Compose profiles to run different combinations of services:
+| Profile           | Command                                               | Services                              |
+| ----------------- | ----------------------------------------------------- | ------------------------------------- |
+| **Full Local**    | `docker-compose --profile local up`                   | Frontend + Backend + Database + Queue |
+| **Frontend Only** | `docker-compose --profile frontend up`                | Next.js app + MongoDB                 |
+| **Backend Only**  | `docker-compose --profile backend-local up`           | API + Redis + Celery                  |
+| **AI Servers**    | `docker-compose --profile servers up`                 | GPU-powered AI services               |
```

Comment:
Do you think we should keep this one as the AI servers are in another repository now?